swift/deepgram-transcribe-audio/README.MD

```diff
@@ -0,0 +1,129 @@
+# Transcribe Audio Files
+
+A Swift Cloud Function that transcribes an Audio file with the help of Deepgram.
```

Comment:
with the help of [Deepgram API](https://deepgram.com/)

Reply:
Done!